content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -168,16 +177,34 @@ topologyKey: kubernetes.io/hostname
 whenUnsatisfiable: ScheduleAnyway
 ```
 
+{{% notice note %}}
+topologyKey refers to the key of KubeVirt infra node labels.
+{{% /notice %}}
+
 this allows us to spread Virtual Machine equally across a cluster.
 
+**Advanced settings: customize Topology Spread Constraints and Node Affinity**
+
 However, it is possible to change the default behaviour and create your own topology combined with Node Affinity Presets.
 You can do it by expanding *ADVANCED SCHEDULING SETTINGS* on the initial nodes dashboard page.
 
 ![Instance Types and Preferences](/img/kubermatic/main/architecture/supported-providers/kubevirt/scheduling-form.png?classes=shadow,border "Advanced Scheduling Settings")
 
+{{% notice note %}}
```

Comment:
I would make a one note at the beginning of this document (after [this](https://github.com/kubermatic/docs/pull/1398/files#diff-0635e8036d60226b7f31bab3900a9b4a1042b68f7495ed9896499299a3538468R168))  that topology spread constraints and node affinity presets are applicable to KubeVirt infra nodes. Or instead of doing a note I would rebuild introduction to highlight that.